### DISCUSSION TITLE
How to obtain the signal strength for a specified coordinate in the coverage map

### DISCUSSION THREAD

**Discussion by vcyycv0** [NVlabs/sionna#469] [Q&A] [answered]
When I set the transmit antenna array in a map and use “scene.coverage_map” . how can I get the signal strength for a specified coordinate in the map?

**Comment by SebastianCa** [comment, marked as answer]
Hi @vcyycv0,

you can access the coverage map at a specific position via the [as_tensor()](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.CoverageMap.as_tensor) method.
Please have a look at the [Sionna Ray Tracing Tutorial](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#Coverage-Map) for further details.
